nixos/doc/manual/development/option-declarations.section.md

```diff
@@ -27,9 +27,10 @@ The function `mkOption` accepts the following arguments.
 
 `type`
 
-:   The type of the option (see [](#sec-option-types)). It may be
-    omitted, but that's not advisable since it may lead to errors that
-    are hard to diagnose.
+:   The type of the option (see [](#sec-option-types)). This
+    argument is mandatory for nixpkgs modules. Setting this is highly
+    recommended for the sake of documentation and type checking. In case it is
+    not set, a fallback type with unspecified behavior is used.
```

Comment:
> unspecified behavior

Wait what? Surely we can specify what the type is, or will be?

Reply:
Ok, I can get on board with that.
Instead of `anything` I'd go with `raw` though, as it does not sneak extra arbitrary decisions into underspecified code. `anything` may be better than `unspecified`, but it is still arbitrary.